internal/fakecgo/symbols.go

```diff
@@ -1,3 +1,4 @@
+// THIS FILE IS GENERATED BY 'go generate' with gen.go. DO NOT EDIT!!!
```

Comment:
Let's use the standardized comment (see https://pkg.go.dev/cmd/go#hdr-Generate_Go_files_by_processing_source):

```
// Code generated by 'go generate' with gen.go. DO NOT EDIT.

// SPDX-License-Identifier: Apache-2.0
// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
```